source/retryable-reads/tests/gridfs-download-serverErrors.yml

```diff
@@ -0,0 +1,152 @@
+database_name: &database_name "retryable-reads-tests"
+collection_name: &collection_name "files"
+
+data:
+    files:
+        -
+            _id: { "$oid" : "000000000000000000000003" }
+            length: 2
+            chunkSize: 4
+            uploadDate: { "$date" : "1970-01-01T00:00:00.000Z" }
+            md5: "c700ed4fdb1d27055aa3faa2c2432283"
+            filename: "length-2"
+            contentType: "application/octet-stream"
+            aliases: []
```

Comment:
According to the [GridFS spec](https://github.com/mongodb/specifications/blob/master/source/gridfs/gridfs-spec.rst), `md5`, `contentType`, and `aliases` are all deprecated. I think it makes sense to omit them. AFAIK, no driver is required to use these fields and I'm not aware of any that relied on `md5` to validate files during read operations.